tests/phpunit/tests/html-api/generate-html5-named-character-references.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+require_once __DIR__ . '/../../../../src/wp-includes/class-wp-token-map.php';
+
+/**
+ * Stores a mapping from HTML5 named character reference to its transformation metadata.
+ *
+ * Example:
+ *
+ *     $entities['&copy;'] === array(
+ *         'codepoints' => array( 0xA9 ),
+ *         'characters' => '©',
+ *     );
+ *
+ * @see https://html.spec.whatwg.org/entities.json
```

Comment:
I was going to suggest something similar. Maybe the entities data file should go in a directory with a README that can links to https://html.spec.whatwg.org/entities.json.

Reply:
I've added a comment. Can leave it open to where it belongs, as I had almost zero sense of where that should be.

I don't like it in the `src` directory because it isn't source code in the same sense. Some test directory?

Reply:
I think the data file is in the right place. My recommendation was something like this:

- tests/phpunit/data/html5-entities/entities.json <- the file
- tests/phpunit/data/html5-entities/README.md <- explanation

This is similar to https://github.com/WordPress/wordpress-develop/tree/19faad774454e86acdd1f0549816cff22b0eb9d8/tests/phpunit/data/html5lib-tests that has a readme with relevant information about the data included.

Reply:
Sounds like a good plan. You can also reference the README in the script that generates the file.

Reply:
README added and directory restructured.